### PR TITLE
Review my own project

### PR DESCRIPTION
**Pull request by mryeminthu**
- Add html file
- Add css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<body>
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+    </div>
+    <div id="tribute-info"></div>
+</div> 
+
```
Comment (mryeminthu) [inline index.html:19]: Please use HTML5 semantics tags (for example, header, nav) on the divs. Some benifits of using these tags are :

- Provide your page with a Search Engine Optimization (SEO) score and
- Ease code navigation when updating features or troubleshooting bugs

**Review by mryeminthu — COMMENTED** [review]
STATUS: CHANGE REQUIRED ♻️

Hi @mryeminthu ,

Good job with this project so far! But there are still some which aren't done yet.

Please kindly use meaningful commit messages. Commit messages are necessary to communicate between team members. Good commit message should have following qualities:

- It should be understandable even by a glance to the header of the commit message.
- It should be short enough, and not too detailed.
- It should be clear enough to understand. 
- It should be capitalized the first word and do not end in punctuation.

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @mryeminthu, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 

## Required Changes ♻️

- [ ] Very well done 👏🏻 , you did a great job by giving a code review yourself 🚀  in a professional manner but you haven't fulfilled the project requirement. Please adhere to the project requirement according to which you should have to make at least 2 inline comments on `HTML/CSS` best practices but you add only one inline comment. Kindly make two inline comments apart from the **PR code review** summary 📔. **Suggestion:** Make one at least comment on HTML and at least one comment on CSS.

## Project Requirement
Screenshot from 2023-06-05 12-20-29

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by skyv26** [comment]
@mryeminthu Please do code review again for yourself with summary + 2 inline comments 👍🏻 .

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <div id="tribute-info"></div>
+</div> 
+   
+    <div class="" id="img-div">
+        <img
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          id="image"
+        />
+
+    <!-- <div class="" id="img-div">
```
Comment (mryeminthu) [inline index.html:26]: - Please refrain from adding unused codes. Unused code like comments make your code messy.

**Review by Cabraham1 — APPROVED** [review]
Hi @mryeminthu  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**